pdal/PipelineManager.cpp

```diff
@@ -53,6 +53,14 @@ PipelineManager::PipelineManager(point_count_t streamLimit) :
     m_progressFd(-1), m_input(nullptr)
 {}
 
+PipelineManager::PipelineManager(SimplePointTable& pointTable,
+                                FixedPointTable& fixedPointTable) :
+    m_factory(new StageFactory),
```

Comment:
Gee, while we're doing this, I wonder if we shouldn't let the user provide their own StageFactory too.